### ISSUE TITLE
deliverInboundMessages cranks cost as much as 2M computrons in agorictest-17

### ISSUE THREAD

**Issue by dckc** [Agoric/agoric-sdk#3760]
## Describe the bug

blocks 62371 to 62382 of agorictest-17 were quite busy.

They include some surprisingly expensive vattp deliveries such as:

```json
{"time":1629839153.2709339,"type":"deliver","crankNum":1636995,"vatID":"v14","deliveryNum":520843,"kd":["message","ko33",{"method":"deliverInboundMessages" ...
...
{"time":1629839153.5546927,"type":"deliver-result","crankNum":1636995,"vatID":"v14","deliveryNum":520843,"dr":["ok",null,{"meterType":"xs-meter-10","compute":2003781,"allocate":79822880}]}
```

## Additional context

slog excerpt:

[block62371-62382.slog.gz](https://github.com/Agoric/agoric-sdk/files/7043726/block62371-62382.slog.gz)

timing diagram rending:

https://gist.github.com/dckc/c74639b2c2afb0650d76b4366cf07ee6

rendered using:

https://github.com/Agoric/agoric-sdk/blob/slog-to-causeway/packages/SwingSet/tools/slog-to-diagram.mjs 33b99c4

**Comment by dckc** [comment]
@mhofman did we have any loadgen clients doing business around block block 62372 at 2021-08-24T21:04:48Z?

**Comment by mhofman** [comment]
Nope, nothing running since 23rd where I ran some loadgen for about 30 minutes
